sys-cluster/kube-router/kube-router-0.4.0.ebuild

```diff
@@ -0,0 +1,160 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+EGO_PN="github.com/cloudnativelabs/${PN}"
+
+inherit golang-build golang-vcs-snapshot
```

Comment:
Is *golang-vcs-snapshot* used here? I am not familiar with go ebuilds, but looks like it downloads an archive...

Does this work with go-module.eclass?

Reply:
Upstream switched to go-modules after that release, without `go.mod` but with flags enabled by `go-module.eclass` `go` will try to download and vendor modules.

golang-vcs-snapshot basically helps to unpack the tarball so that I can later use `GOPATH=$S` so `go` can find the package and its dependencies